docker/Dockerfile.customopbuilder.clean

```diff
@@ -0,0 +1,92 @@
+FROM tensorflow/tensorflow:custom-op-gpu as base
+
+ARG PYVER=2.7
+ENV PYVER=${PYVER}
+ARG PYV=27
+ENV PYV=${PYV}
+ARG USE_CUDA_VERSION=10
+ENV USE_CUDA_VERSION=${USE_CUDA_VERSION}
+
+RUN if [ "${USE_CUDA_VERSION}" = "9" ]; then \
```

Comment:
Maybe you can reuse CUDA package from https://github.com/NVIDIA/DALI/blob/master/Dockerfile.deps?
Now we have to maintain it in two places and it will soon get out of sync.

Reply:
I'll try

Reply:
done